Source/ACE.Entity/AceObject.cs

```diff
@@ -973,6 +973,34 @@ public object Clone()
             return ret;
         }
 
+        public object CopyClone(uint guid)
```

Comment:
I don't understand the distinction between CopyClone and clone.  Why do we need two separate functions, and what is the difference?